include/seqan3/io/all.hpp

```diff
@@ -100,7 +100,7 @@
 
 #pragma once
 
-#include <seqan3/io/alignment_file/all.hpp>
+#include <seqan3/io/sam_file/all.hpp>
 #include <seqan3/io/exception.hpp>
 #include <seqan3/io/record.hpp>
```

Comment:
Falsche Reihenfolge:

```suggestion
#include <seqan3/io/exception.hpp>
#include <seqan3/io/record.hpp>
#include <seqan3/io/sam_file/all.hpp>
```